src/vehicle_type.h

```diff
@@ -56,7 +56,7 @@ static const VehicleID INVALID_VEHICLE = 0xFFFFF; ///< Constant representing a n
 /** Pathfinding option states */
 enum VehiclePathFinders {
 	// Original PathFinder (OPF) used to be 0
-	VPF_NPF  = 1, ///< New PathFinder
+	// New PathFinder (NPF) used to be 1
 	VPF_YAPF = 2, ///< Yet Another PathFinder
```

Comment:
Is this still referenced anywhere now that it's the only option?

Reply:
Given the whole choice for pathfinders has been completely ripped out instead of just the NPF case, this enumeration won't be used so it ought to be removed as well I'd say.

Alternatively the setting for pathfinder is not removed, but forced to YAPF, and the original switches are returned without the NPF case. That would make introducing a new pathfinder easier, but how likely is that given YAPF is almost 18 years old and NPF is just over 19 years old. 

Given OpenTTD is just over 20 years old, the pathfinders were all introduced in the first ~10% of its lifetime and after that it's been silent. So a new pathfinder seems to be quite unlikely, and as such I'd vote for just removing the enumeration.